website/docs/r/one_dashboard_json.html.markdown

```diff
@@ -107,6 +107,396 @@ resource "newrelic_one_dashboard_json" "bar" {
   "variables": []
 }
 ```
+### Dashboard With Multiple Pages
+
+The following example demonstrates creating a dashboard with multiple pages, with each page comprising a couple of widgets.
+
+`dashboard.json`
```

Comment:
Can you include what the resource would look like?

Reply:
[Added a HCL block](https://github.com/newrelic/terraform-provider-newrelic/pull/2347/files#diff-3c4c21011e01919758a5262f44acd65c06b6c4f35b259806978a2e637d342790R113-R117) in the most recent commit.